src/AspNet.Security.OAuth.WorkWeixin/AspNet.Security.OAuth.WorkWeixin/WorkWeixinAuthenticationExtensions.cs

```diff
@@ -0,0 +1,83 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AspNet.Security.OAuth.WorkWeixin;
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Extension methods to add Weixin authentication capabilities to an HTTP application pipeline.
```

Comment:
I guess all references to "Weixin" in these comments should actually be "Work Weixin"?

Reply:
Or depending on which is correct, "WeCom".